Source/Chronozoom.Entities/Chronozoom.Entities.csproj

```diff
@@ -171,6 +171,7 @@
     <Content Include="SQL\ManualMigration\201408130000000_MultipleCollections.sql" />
     <Content Include="SQL\ManualMigration\201409250000000_PubliclySearchableChange.sql" />
     <Content Include="SQL\ManualMigration\201502100000000_AddBackgroundUrlAspectRatio.sql" />
+    <Content Include="SQL\ManualMigration\201502170000000_AddYOffsetsAttributes.sql" />

```

Comment:
Where is 201502160000000_AddExhibitYOffsetAttribute.sql that I saw in your database?